lib/providers/syntheticStatusProvider.ts

```diff
@@ -36,7 +36,7 @@ export class UPASyntheticStatusProvider implements SyntheticStatusProvider {
       );
 
       return {
-        useSynthetic: result.data.useSynthetic,
+        useSynthetic: result.data.enabled,
```

Comment:
```suggestion
        syntheticEnabled: result.data.enabled,
```
nit, just cause we use `useSynthetic` for the request flag i think?